webview.go

```diff
@@ -54,16 +54,20 @@ type webview struct {
 }
 
 // New creates a new webview in a new window.
-func New(debug bool) WebView { return NewWindow(debug, nil) }
+func New(debug bool) WebView { return NewWindow(debug, "", nil) }
+
+// NewWithDataPath creates a new webview in a new window.
+func NewWithDataPath(debug bool, dataPath string) WebView { return NewWindow(debug, dataPath, nil) }
```

Comment:
Could we do `NewWithOptions` and have an options struct instead? We still need `New` and `NewWindow` to take a `debug` option for backwards compatibility, but it would be nice if the new struct had both the `Debug` and`DataPath` options, and the old function just forwarded `debug` to it. In a future API break we could consider removing the legacy `debug` parameter.